src/steamship/app/response.py

```diff
@@ -183,7 +183,7 @@ def from_obj(obj: Any) -> Response:  # noqa: C901
                 logging.error(f"Failed calling to_dict on response object. {obj}\n {e}")
 
         if isinstance(obj, BaseModel):
-            return Response(json=obj.dict())
+            return Response(json=obj.dict(by_alias=True))
```

Comment:
@EniasCailliau just want to call out this change -- you OK with this?

Reply:
If all goes well, we don't need to call `by_alias` here. The Response's init should pass by Line 73 of `binary_utils.py` and convert it for you in the camelCase format for you: 

```
elif isinstance(json, CamelModel):
    ret_dict = json.dict(by_alias=True)
```

Reply:
sg. reverting!